[us1.prod]JasSpencer/Added small note to _index.md (#16148)
DataDog/documentation: content/en/monitors/configuration/_index.md

In this case, you should enable notifications for missing data. The sections below explain how to accomplish this with each option.

> **Note**: Alerting on missing data only works when data is first present. The monitor must be able to evaluate before alerting on missing data. For example, creating a monitor for `service:abc` when data from that `service` has not yet reported to Datadog will result in no alerts.

Reviewer: ```suggestion
**Note**: The monitor must be able to evaluate data before alerting on missing data. For example, if you create a monitor for `service:abc` and data from that `service` is not reporting, the monitor does not send alerts.
```
